incuna-sass/mixins/_svg.sass

```diff
@@ -0,0 +1,12 @@
+//////////  S V G  M I X I N  //////////////////////////////
+// To use like you would a sprite with the added benefit that
+// it can be whatever size you like and will work perfectly
+// on retina screens
+@mixin svg($file, $width: 100%, $height: $width)
+    @include background(image-url('svg/#{$file}.svg'))
+    background:
+        repeat: no-repeat
+        size: contain
+    width: $width
+    height: $height
+    display: inline-block
```

Comment:
but you'll have to set a display value most of the time (depending what element you put it on, but considering we use pseudo elements for this sort of thing a lot) having one by default makes sense to me. I actually added it because not having it was more annoying than having it be the wrong thing. Our icon-font mixin has `display: inline-block` too. I think if it set `display: block` we'd be overriding it more than we will with `display: inline-block`. If it turns out not to be useful we can always take it out later. Sometimes you just have to see what happens, you can't future proof everything :D